### PR TITLE
Examples adapted from vOptGeneric

### PR DESCRIPTION
**Pull request by xgandibleux**

### CODE REVIEW

**Comment by odow** [comment]
I've added these examples directly to the JuMP documentation: https://github.com/jump-dev/JuMP.jl/pull/3176/commits/e0a9438a13a69af5fc97f8d75226dff7f3ace52d. That should give them a wider audience :smile:

Once the JuMP PR is merged, we could add a link to this README directing people to the JuMP docs?

File: examples/bilap.jl [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+@expression(bilap, objective2, sum( C2[i,j]*x[i,j] for i=1:n,j=1:n ))
+@objective(bilap, Min, [objective1, objective2])
+@constraint(bilap , cols[i=1:n], sum(x[i,j] for j=1:n) == 1 )
+@constraint(bilap , rows[j=1:n], sum(x[i,j] for i=1:n) == 1 )
+
+
+# ---- Invoking the algorithm (Epsilon Constraint method) and the IP solver (GLPK) 
+set_optimizer(bilap, () -> MOA.Optimizer(GLPK.Optimizer))
+set_optimizer_attribute(bilap, MOA.Algorithm(), MOA.EpsilonConstraint())
+set_optimizer_attribute(bilap, MOA.ObjectiveAbsoluteTolerance(1), 1) # ugly name for this attribute
```
Comment (odow) [inline examples/bilap.jl:42]: Should we make this the default for `EpsilonConstraint`, so that in most cases users don't have to set this attribute?

**Comment by odow** [comment]
Here's the link: https://jump.dev/JuMP.jl/previews/PR3176/tutorials/linear/multi_objective_examples/

**Comment by xgandibleux** [comment]
A proposal to reach a wide audience with the MOA tutorial; I would suggest to introduce the example on the knapsack without using a structure, nor the broadcasting operator, just using simple vectors and matrices. It simplify the syntax the example, making it more close with the mathematical description, and thus facilitate the understanding for the users coming from mathlab, and/or beginners with Julia.

**Pull request closed without merging**